rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -788,6 +795,47 @@ stream {
         {{ end }}
 
         location {{ $path }} {
+            {{ if $location.LuaRestyWAF.Enabled }}
+            access_by_lua_block {
+                local lua_resty_waf = require("resty.waf")
+                local waf = lua_resty_waf:new()
+
+                waf:set_option("mode", "ACTIVE")
+                waf:set_option("allowed_content_types", { "text/html", "text/json", "application/json" })
```

Comment:
in a subsequent PR I'll add an annotation to override this also will introduce another annotation to configure custom ruleset